app/javascript/pages/Emails/Scheduled.tsx

```diff
@@ -269,6 +278,55 @@ export const ScheduledTab = () => {
           />
         )}
       </div>
-    </Layout>
+    </EmailsLayout>
+  );
+}
+
+const ViewEmailButton = (props: { installment: SavedInstallment }) => {
+  const [sendingPreviewEmail, setSendingPreviewEmail] = React.useState(false);
+
+  return (
+    <Button
+      disabled={sendingPreviewEmail}
+      onClick={asyncVoid(async () => {
+        setSendingPreviewEmail(true);
+        try {
+          await previewInstallment(props.installment.external_id);
+          showAlert("A preview has been sent to your email.", "success");
+        } catch (error) {
+          assertResponseError(error);
+          showAlert(error.message, "error");
+        } finally {
+          setSendingPreviewEmail(false);
+        }
+      })}
+    >
+      <Icon name="envelope-fill"></Icon>
+      {sendingPreviewEmail ? "Sending..." : "View email"}
+    </Button>
   );
 };
+
+const EmptyStatePlaceholder = ({
+  title,
+  description,
+  placeholderImage,
+}: {
+  title: string;
+  description: string;
+  placeholderImage: string;
+}) => (
+  <Placeholder>
+    <figure>
+      <img src={placeholderImage} />
+    </figure>
+    <h2>{title}</h2>
+    <p>{description}</p>
+    <NewEmailButton />
+    <p>
+      <a href="/help/article/169-how-to-send-an-update" target="_blank" rel="noreferrer">
+        Learn more about emails
+      </a>
+    </p>
+  </Placeholder>
+);
```

Comment:
Must have missed this before, but those two components are duplicated across `Drafts`, `Published` and `Scheduled`, please export them to their own dedicated component files and import them to the Inertia pages